src/infra/postgres/client/client.go

```diff
@@ -120,3 +132,24 @@ func (c PostgresClient) RunInTransaction(ctx context.Context, persist func(clien
 
 	return c.db.(*pg.DB).RunInTransaction(ctx, persistFunc)
 }
+
+func (c *PostgresClient) SelectMany(ctx context.Context, model, dst interface{}, condition string, params ...interface{}) error {
+	q := c.db.ModelContext(ctx, model)
+	err := q.Where(condition, params...).Select(dst)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (c *PostgresClient) Ping(ctx context.Context) error {
+	return c.pgdb.Ping(ctx)
+}
+
+func (c *PostgresClient) CreateTable(ctx context.Context, model interface{}) error {
+	return c.db.ModelContext(ctx, model).CreateTable(nil)
+}
+
+func (c *PostgresClient) DropTable(ctx context.Context, model interface{}) error {
+	return c.db.ModelContext(ctx, model).DropTable(nil)
```

Comment:
This cannot exist in this client as it should never be needed in the app.

Reply:
We decided to keep it in the review finally until we use real migrations even for tests.

Reply:
Sorry, dO you mind the to detail why you need this? Our migration are tested as part of the integration test I don't see a place for this either